guide/daily-spending-wallet/backup-and-recovery/manual-backup.md

```diff
@@ -21,64 +21,39 @@ Illustration sources:
+   layout = "full-width"
 %}
 
 # Manual backup
 
-## Backing up channel state
-
-A channel state backup is typically a file which includes information about the user's channel balances. This file can be downloaded onto the user's device and used to restore the Lightning channel state on another device. This could be a database of the most current channel balance information, or a static backup (as an absolute last resort).
-
-{% include tip/open.html color="red" icon="forbid" label="Manual channel state backups are for advanced users" %}
+The daily spending wallet described in this reference design is designed to make backup as easy as possible for the user. As a result, the default flow is one where the user opts-in to a cloud backup.
 
-Manual backups of Lightning channel state should only be performed with careful consideration. This is because the channel state backup may become outdated very quickly, and attempting to restore from an outdated backup could harm the user's funds. In the interest of adhering to the principles of [self-custody]({{ '/guide/getting-started/principles/#self-custody' | relative_url }}) and [transparency]({{ '/guide/getting-started/principles/#transparency' | relative_url }}), you may choose to allow your user to perform a manual channel state backup. However, this is not recommended for the majority of users. If this option is available, keep it tucked away in a settings menu and prepend a warning that the option is only for advanced users.
+However, some users may not be comfortable with the idea of their private key on a cloud server. So in accordance with the design principles of [self-custody]({{ '/guide/getting-started/principles/#self-custody' | relative_url }}) and [transparency]({{ '/guide/getting-started/principles/#transparency' | relative_url }}), this wallet also offers a manual backup option.
 
-{% include tip/close.html %}
+## A note about channel state
 
-## Backing up wallet recovery data
+The lightning channel state is subject to change frequently, particularly if the user truly uses the daily spending wallet every day as the name would imply. As a result, we can’t think about the channel state as being a one-time backup. Channel state must be backed-up everytime the state is updated.
 
-Manual backup should be used primarily for wallet recovery data. Wallet recovery data includes:
-- The [recovery phrase]({{ '/guide/glossary/#recovery-phrase' | relative_url }})
-  - Type - BIP39, Electrum, Shamir Shares (SLIP39), AEZEED
-  - Passphrase (optional)
-- Derivation path
-- Fingerprint
+As a result, this wallet requires channel state to be backed up automatically to a cloud provider. Allowing for it to be backed up manually would create a false sense of security.
 
+This trade-off is designed to protect the user. In the event that a 3rd party manages to obtain and decrypt the channel state from the cloud storage, there is not a lot they can do; they have no way of stealing the funds without the keys.
```

Comment:
```suggestion
This trade-off is designed to protect the user. In the event that a 3rd party manages to obtain and decrypt the channel state from the cloud storage, there is not a lot they can do; they have no way of stealing the funds without the recovery phrase.
```

- Which trade-off exactly? Wasn't super clear reading this
- I thought it was optionally encrypted? If it is encrypted, what by? (not mentioned in cloud backup page).

Reply:
When a manual backup is made the state backups should be encrypted by the recovery phrase. As you need the recovery phrase to recover your wallet this adds extra security without any additional friction.

Reply:
Changed it to "requirement", not sure why I worded it "trade-off".

Reply:
Yeah, that would make sense to encrypt with the recovery phrase, I think. There's not really much that can even be done with the channel state if the user lacks the recovery phrase, so it makes sense to do it this way.

Reply:
Changed it to talk about encrypting with the recovery phrase

Reply:
Sweet, LGMT. Yeah its mostly a privacy benefit but it adds no additional friction for users so why not